app/Http/Controllers/Event/EventController.php

```diff
@@ -109,11 +111,20 @@ public function update(EventRequest $request, $id)
 	}
 
 	/**
-	 * Cancel an event.
+	 * Organization cancel an event
+	 */
+	public function cancel($id)
+	{
```

Comment:
You must check here that the authenticated organization is the one who created the event